concourse/scripts/terraform_infra.bash

```diff
@@ -0,0 +1,44 @@
+set -e
+set -x
+
+apk add --no-cache jq
+
+aws_account_id=$(echo "$tat_secrets_ui" | jq -r .aws_account_id)
+aws_access_key_id=$(echo "$tat_secrets_ui" | jq -r .aws_access_key_id)
+
+aws_secret_access_key=$(echo "$tat_secrets_ui" | jq -r .aws_secret_access_key)
+api_secret_name=$(echo "$tat_secrets_ui" | jq -r .api_secret_name)
+
+domain=$(echo "$tat_secrets_ui" | jq -r .domain)
+container_ver=$(cat resource-repo/tag)
+
+api_bucket_name=$(echo "$tat_secrets_ui" | jq -r .api_bucket_name)
+ui_secret_name=$(echo "$tat_secrets_ui" | jq -r .ui_secret_name)
+
+aws_account_name=$(echo "$tat_secrets_ui" | jq -r .aws_account_name)
+localhost=$(echo "$tat_secrets_ui" | jq -r .localhost)
+
+container_image=$(echo "$tat_secrets_ui" | jq -r .container_image)
+force_deployment=$(echo "$tat_secrets_ui" | jq -r .force_deployment)
```

Comment:
Bash sucks at parsing JSON, was thinking of rewriting this chunk in Python. Thoughts?